### ISSUE TITLE
[BUG] Security Vulnerability

### ISSUE THREAD

**Issue by pakesson-truesec** [mlflow/mlflow#8273]
Details sent to `mlflow-oss-maintainers@googlegroups.com`.

**Comment by BenWilson2** [comment]
Thank you for the report @pakesson-truesec . We're looking into fixes!

**Comment by BenWilson2** [comment]
Hi @pakesson-truesec thank you for bringing this to our attention. Once #8281 is merged to master, could you perform a validation on your end to let us know if there are any additional creative ways that you can find that our changes didn't cover? 
Thank you for helping to make the project more secure! It is greatly appreciated by both the maintainers and our users.

**Comment by BenWilson2** [comment]
Hi @pakesson-truesec the fix for the reported vulnerability has just merged into master.

**Comment by pakesson-truesec** [comment]
Great, I just tested it and it seems to cover my cases!

**Comment by harupy** [comment]
@pakesson-truesec Thanks for the confirmation :)
